lib/completion.dart

```diff
@@ -57,6 +58,24 @@ class DartCompleter extends CodeCompleter {
             ));
           }
         }
+      });
+      var assistsComplete =
+          servicesApi.assists(request).then((AssistsResponse response) {
+        for (var assist in response.assists) {
+          var sourceEdits = <SourceEdit>[];
+          for (var edit in assist.edits) {
+            sourceEdits
+                .add(SourceEdit(edit.length, edit.offset, edit.replacement));
+          }
+          var completion = Completion('',
```

Comment:
What's the first arg to `Completion` (and, why's it empty)?

Reply:
It's the `value` argument. It's not being here or for fixes because quick fixes can have multiple SourceEdits. This value ends up being set to `HintResult.text` which will also be applied, but since we have multiple edits it doesn't need to be set.

Not sure if I'm explaining that accurately, but that's my understanding from reading the `_completionHelper()` function in `editor_codemirror.dart`

Reply:
Thanks; doesn't sounds like this this call site is an issue, but likely the actual API of the `Completion` constructor could be improved.